app/assets/stylesheets/application.css

```diff
@@ -24,4 +24,5 @@
  *= require comments
  *= require wiki
  *= require question
+ *= require swagger-ui
```

Comment:
Sorry--I was pulling them out as I went, and wasn't sure about this one--I'll take it out.